Jenkinsfile.nightly

```
@@ -405,10 +387,10 @@ void sendPipelineErrorNotification() {
                 to: env.KOGITO_CI_EMAIL_TO
 }
 
-List getDefaultBuildParams(String buildBranchName = '') {
+List getDefaultBuildParams(String buildBranchName = '', String key = '') {
     buildBranchName = buildBranchName ?: getBuildBranch()
     List params = []
-    addStringParam(params, 'DISPLAY_NAME', env.NIGHTLY_TAG)
+    addStringParam(params, 'DISPLAY_NAME', "${key ? key + '-' : ''}${env.NIGHTLY_TAG}")
```

```suggestion
    addStringParam(params, 'DISPLAY_NAME', "${key ? "${key}-" : ''}${env.NIGHTLY_TAG}")
```

You can try 
```
def job = 'jobvalue'
def key = 'key1'

println "${job ? "${key}-" : 'aaa'}"
```
at https://groovyconsole.appspot.com/ to check string replacement is properly working

---

I wasn't sure about that. Thanks for the link !!! :)